package/features/steps/cli_steps.py

```diff
@@ -86,6 +87,19 @@ def install_project_requirements(context):
         assert False
 
 
+@given("I have installed the lower-bound Kedro-viz requirements")
+def install_lower_bound_requirements(context):
+    cwd = Path(__file__).resolve().parent
+    requirements_path = cwd / "lower_requirements.txt"
```

Comment:
should we be updating the lower_requirements.txt everytime manually ? If so, will it be helpful to add somewhere in the docs when to do so ?

Reply:
good idea.